src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -63,55 +113,49 @@ export class ResultsViewPageStore {
         else {
             const store = new MutationMapperStore(AppConfig,
```

Comment:
@adamabeshouse @onursumer this is not related to this review but i've been wanting to discuss, the collection of MutationMapperStores is not "reactive" because they get instantiated by view layer calling this method.  This became a problem for loading the MutEx tab prior to the Mutations tab because these stores had not yet been instantiated.  I am ended up just instantiating them on that tab and I do see that they will be resused when switching tabs, but still, I think we should consider a different solution.  Lets discuss soon.

Reply:
you're right -- it should be a computed map or a computed flat list or something - something reactive